source/index.rst

```diff
@@ -8,11 +8,124 @@ Welcome to the MinIO Documentation! MinIO is a high performance object storage
 solution with native support for Kubernetes deployments. MinIO provides an 
 Amazon Web Services S3-compatible API and supports all core S3 features. 
 
-First-time users of MinIO *or* object storage services should start with 
-our :doc:`Introduction </introduction/minio-overview>`.
+You can get started exploring MinIO features using our ``play`` server at
+https://play.min.io. ``play`` is a *public* MinIO cluster running the latest
+stable MinIO server. Any file uploaded to ``play`` should be considered public
+and non-protected.
 
-Users deploying onto a Kubernetes cluster should start with our 
-:docs-k8s:`Kubernetes documentation <>`.
+The MinIO Client :mc:`mc` commandline interface includes an
+:mc-cmd:`alias <mc alias>` for the ``play`` server. After 
+`Downloading the MinIO Client <https://min.io/downloads>`__, use the 
+``play`` alias to perform S3-compatible object storage operations:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc alias list play
+   mc mb --with-lock play/mynewbucket
+   mc cp ~/data/mytestdata play/mynewbucket
+
+See the :doc:`MinIO Client Complete Reference </reference/minio-cli/minio-mc>`
+for complete documentation on the available :mc:`mc` commands.
+
+- First-time users of MinIO *or* object storage services should start with 
+  our :doc:`Introduction </introduction/minio-overview>`.
+
+- Users deploying onto a Kubernetes cluster should start with our 
+  :docs-k8s:`Kubernetes documentation <>`.
+
+Quickstart
+----------
+
+The following steps deploys MinIO in filesystem mode with a single folder or
+disk on the local host. This deployment is best used for initial evaluation of
+MinIO S3-compatible object storage. Filesystem mode does not support features
+such as versioning and replication.
+
+1\) Create a Data Folder
+   Create a folder on the local drive for MinIO to use for object storag
+   operations. For example:
+
+   .. code-block:: shell
+      :class: copyable
+
+      mkdir /mnt/data
+      sudo chmod -R 775 /mnt/data
+      
+2\) Download MinIO Server and Commandline Tools
+   Visit `https://min.io/downloads <https://min.io/downloads?ref=docs>`__ and
+   following the instructions for your host operating system to download and
+   configure the :mc:`minio` and :mc:`mc` binaries.  Consider adding the
+   ``minio`` and ``mc`` binaries to the operating system PATH for simplified
+   operations.
+
+   Follow the instructions on the download site to start the :mc:`minio server`
+   process. For example, the following command starts the
+   :mc:`minio server` using the created directory:
+
+   .. code-block:: shell
+      :class: copyable
+
+      export MINIO_ROOT_USER_FILE=myminioaccesskey
+      export MINIO_ROOT_PASSWORD_FILE=myminiosecretkey
+      export MINIO_KMS_SECRET_KEY=myminioencryptionkey
```

Comment:
This will not work. The server will fail to parse that value:
Either we specify a concrete value - like:
```sh
 export MINIO_KMS_SECRET_KEY="my-encryption-key:JTLb2Nfvgoh+DSi06d1dqcTINXHqvTr2RqlzjcV2jUA="
```
or explain how the generate one  - e.g. `cat /dev/urandom | head -c 32 | base64 -`

Reply:
Right, this was a placeholder I did not replace. Fixed.